src/renderer/components/Icon.vue

```diff
@@ -436,12 +448,30 @@ export default {
     }
   }
 
+  .double-arrow-left {
+    path:first-child {
```

Comment:
Let’s use the class instead of element.

Reply:
sure - was copying previous implementation for consistency. The 'arrow-right' and 'arrow-left' use the element.